pkg/discovery/worker/container_spec_metrics_collector.go

```diff
@@ -48,7 +48,7 @@ func (collector *ContainerSpecMetricsCollector) CollectContainerSpecMetrics() []
 		if util.HasController(pod) {
 			controllerUID, err := util.GetControllerUID(pod, collector.metricsSink)
 			if err != nil {
-				glog.Errorf("Error getting controller UID from pod %s, %v", pod.Name, err)
+				glog.V(3).Infof("Cannot find controller UID for pod %s/%s, %v", pod.Namespace, pod.Name, err)
```

Comment:
I noticed `util.GetControllerUID` func is used in `container_dto_builder` and `pod_entity_dto_builder` as well. Maybe we can only keep one log as Error (maybe the one in pod_entity_dto_builder) and set others as V(3)

Reply:
Make perfect sense. Raised to `V(3)` in `container_dto_builder`, and leave the one `pod_entity_dto_builder` as error.